src/terminal.rs

```diff
@@ -221,6 +249,10 @@ pub struct TerminalLoggerConfig {
     #[serde(default)]
     pub source_location: SourceLocation,
 
+    /// process ID
+    #[serde(default)]
+    pub processid: ProcessID,
```

Comment:
I think , in this setting, the associated configuration files will be slightly clunky:
For example,
```toml
type = "terminal" 
source_location = "module_and_line" 
level = "debug"

# Inlcude PID in log messages
processid = "process_id"
```

Hence, it may be better to replace the value type of `processid` to `bool`.
This will allow you to write as the following:
```toml
type = "terminal" 
source_location = "module_and_line" 
level = "debug"

# Inlcude PID in log messages
processid = true
```

Reply:
done